admin/jobs/river/river.go

```diff
@@ -342,6 +342,7 @@ func (c *Client) StartOrgTrial(ctx context.Context, orgID string) (*jobs.InsertR
 		UniqueOpts: river.UniqueOpts{
 			ByArgs: true,
 		},
+		MaxAttempts: 5, // override default retries as init org billing job should complete before this if org creation and project deployment were done in single flow
```

Comment:
What is the default? Is it the 3 we set in the client? If so will going from 3  => 5 give enough time?

Reply:
yes default is 3, going to 5 should give around extra 2 mins which should be sufficient IMO. Its just that init job should complete before this. We should add alerts on failed jobs.